From 815adcd3b657917a995cbf5b47ef6fc6da94c01b Mon Sep 17 00:00:00 2001
Message-ID: <815adcd3b657917a995cbf5b47ef6fc6da94c01b.1718966788.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1718966788.git.sharlatanus@gmail.com>
References: <cover.1718966788.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Thu, 13 Jun 2024 11:35:06 +0100
Subject: [PATCH 12/30] gnu: python-pyvo: Update to 1.5.2.

* gnu/packages/astronomy.scm (python-pyvo): Update to 1.5.2.
[arguments]: <#:test-flags>: Remove test constrains as the issue was
resolved upstream.

Change-Id: I6a4e7ac339786b42792970198023a406dd2db0e5
---
 gnu/packages/astronomy.scm | 9 ++-------
 1 file changed, 2 insertions(+), 7 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index d045220d3c..c61f29f082 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -2696,19 +2696,14 @@ (define-public python-pyavm
 (define-public python-pyvo
   (package
     (name "python-pyvo")
-    (version "1.5.1")
+    (version "1.5.2")
     (source
      (origin
        (method url-fetch)
        (uri (pypi-uri "pyvo" version))
        (sha256
-        (base32 "0cm6agaf1gvc5bi95wx6a70ngj9vn95rx78fs59vlrmpww7q2807"))))
+        (base32 "0wcg3jhfwjd9gqs74mw63sgi1yhmgljx1bwk3mxn0p6fv924r8mq"))))
     (build-system pyproject-build-system)
-    (arguments
-     (list
-      ;; Disable one failing test, see
-      ;; <https://github.com/astropy/pyvo/issues/547>.
-      #:test-flags #~(list "-k" "not test_single_table_description")))
     (native-inputs
      (list python-pytest-astropy python-requests-mock python-setuptools-scm))
     (propagated-inputs
-- 
2.41.0

